Linux-Application/DomesdayDuplicator/advancednamingdialog.cpp

```diff
@@ -180,6 +196,56 @@ void AdvancedNamingDialog::updateGui(void)
 
 }
 
+// Update the GUI and hold values from previous side input
+void AdvancedNamingDialog::updateSideHoldings(void)
+{
+    if ((ui->discSideSpinBox->value()) == 2 ) {
+
+        if (mintSide1.length() == 0){
+        mintSide1 = ui->mintLineEdit->text();
```

Comment:
The indentation doesn't match here (and the two `if`s above have spacing problems).